db/config.cc

```diff
@@ -1083,8 +1083,6 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , restrict_future_timestamp(this, "restrict_future_timestamp",liveness::LiveUpdate, value_status::Used, true, "Controls whether to detect and forbid unreasonable USING TIMESTAMP, more than 3 days into the future.")
     , ignore_truncation_record(this, "unsafe_ignore_truncation_record", value_status::Used, false,
         "Ignore truncation record stored in system tables as if tables were never truncated.")
-    , force_schema_commit_log(this, "force_schema_commit_log", value_status::Used, false,
```

Comment:
I think the usual way is to modify an option to `value_status::Unused` instead of deleting it completely. Otherwise, won't the node complain on startup (maybe even fail startup) if conf/scylla.yaml contains unknown option (I'm not sure) -- which may happen when upgrading from older versions?

Reply:
Fixed.